api/providers/views.py

```diff
@@ -402,14 +436,15 @@ def get_renderer_context(self):
             # TODO don't duplicate the above
             auth = get_user_auth(self.request)
             auth_user = getattr(auth, 'user', None)
-            provider = get_object_or_error(PreprintProvider, self.kwargs['provider_id'], self.request, display_name='PreprintProvider')
+            provider = self.get_provider()
             if auth_user and auth_user.has_perm('view_submissions', provider):
                 context['meta'] = {
                     'reviews_state_counts': provider.get_reviewable_state_counts(),
                 }
```

Comment:
Is this available to registrations as well? The `meta.reviews_state_counts` was mentioned on the [list of things that FR had from preprints](https://www.notion.so/Registries-Moderation-Analysis-8b9a80f0fdb04da98a251112f4bf09be#0f55f0f90a61460ba591e5bd421291ce) that they'd likely need here.